api/src/pagination.js

```diff
@@ -34,6 +34,10 @@ export async function applyPagination (
     .find(filter)
     .count()
 
+  const { aum: totalAUM, activity: totalActivity } = (await query.find(filter).toArray())
+    .map(({ aum, activity }) => ({ aum, activity }))
+    .reduce((accum, { aum, activity }) => ({ aum: aum + accum.aum, activity: activity + accum.activity }))
+
```

Comment:
@onbjerg I didn't have clear how to aggregate AUM and Activity into a sum. Is this the way to go, or graphql has other fancy stuff to make it?